themes/openideal_theme/templates/components/login-with.html.twig

```diff
@@ -0,0 +1,16 @@
+{% set module = 'social-auth' %}
+
+{{ attach_library('social_auth/auth-icons') }}
+
+{% for social_network in social_networks %}
+  {% if destination %}
+    <a class="{{ module }} auth-link" href="{{ path(social_network.route) }}?destination={{ destination }}">
```

Comment:
Can we prepare url with all necessary parameters outside of the twig template?

Reply:
I reworked social-auth template. Don't want to change their theme variables and logic. But it necessary I can do this.

Reply:
Please ignore this request, I thought this is a custom template.